### PR TITLE
fix types according with ultradom types

### PR DESCRIPTION
**Pull request by dancespiele**

### CODE REVIEW

File: hyperapp.d.ts [hunk 1]
```diff
@@ -6,48 +6,34 @@ export as namespace hyperapp
  *
  * @memberOf [VDOM]
  */
-export interface VNode<Props> {
-  name: string
-  props: Props
-  children: VNodeChild<object | null>[]
+export interface VNode<Attributes = {}> {
+  nodeName: string
+  attributes?: Attributes
```
Comment (frenzzy) [inline hyperapp.d.ts]: but `attributes` are not optional, they are required, isn't it?
maybe `key` is optional...

Comment (okwolf) [reply hyperapp.d.ts]: I know they can be `null` but I’m not sure if they can be absent entirely.

Comment (jorgebucaran) [reply hyperapp.d.ts]: This is okay.
```js
h(nodeName, null, null)
```

But this is not.

```
{
  nodeName, attributes: null, children: null 
}
```

Comment (jorgebucaran) [reply hyperapp.d.ts]: @dancespiele ☝️

Comment (okwolf) [reply hyperapp.d.ts]: Ah, so the ~~`h`~~ `createNode` function handles that for you?

**Pull request merged**